ember-app/app/services/filters.js

```diff
@@ -11,7 +19,8 @@ var FiltersService = Ember.Service.extend({
     if (!this.get("filterGroups.created")){ return; }
     key = key.replace("Filters", "");
     if (this.get("filterGroups." + key + ".filters")){
-      return this.get("filterGroups." + key + ".filters");
+      return this.set(key+ "Filters", attr(key));
+      return this.get(key + "Filters");
```

Comment:
Huh, good catch. 

Didn't seem to affect anything, the return value of `create` must not be significant